chromium_src/content/browser/renderer_host/render_process_host_impl.cc

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/content/browser/renderer_host/brave_plugin_registry_impl.h"
+
+#define PluginRegistryImpl BravePluginRegistryImpl
```

Comment:
if you are overriding a class like this, you should always include the header for the  class you are overriding before redefining it. Otherwise it's possible that it won't already be defined at this point and you'll change the original header file (likely resulting in a build failure)

Reply:
it gets added from brave_plugin_registry_impl.h, but better to be explicit and make sure there are no issues